designs/extensible-cli-and-scaffolding-plugins-phase-1.md

```diff
@@ -170,8 +170,14 @@ Example Operator SDK main.go:
 
 ```go
 func main() {
+    rootCmdCfg := cli.RootCommandConfig{
```

Comment:
I’d rather the original doc not be modified. PR description and godocs should be sufficient documentation.

Reply:
Makes sense to me